src/Libero/ruleset.xml

```diff
@@ -10,6 +10,12 @@
         <exclude name="PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse"/>
     </rule>
 
+    <rule ref="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses">
+        <properties>
+            <property name="caseSensitive" value="true"/>
```

Comment:
Not sure about this. I like the exampe in https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesalphabeticallysorteduses-, but not what it's done to `RulesetTests`...